Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactSystem.Nodes.cs

```diff
@@ -44,10 +44,10 @@ private void GenerateArtifactNodeTree(EntityUid artifact, ref List<ArtifactNode>
 
     private int GetValidNodeId()
     {
-        var id = _random.Next(10000, 100000);
+        var id = _random.Next(100, 999);
         while (_usedNodeIds.Contains(id))
         {
-            id = _random.Next(10000, 100000);
+            id = _random.Next(100, 999);
```

Comment:
Random.Next's max value is actually an exclusive bound, so I believe this this will only generate between 100-998.